web/src/reducers/admin.js

```diff
@@ -24,3 +26,25 @@ const user = (state = initialState, action) => {
 };
 
 export default user;
+
+export const selectUsers = ({ admin: { users } }) => users;
+
+export const selectUsersSorted = createSelector(
+  [selectUsers],
+  users => {
+    return [...users].sort(
+      ({ name: nameA, email: emailA }, { name: nameB, email: emailB }) => {
+        const sortA = (nameA || emailA).toLowerCase();
+        const sortB = (nameB || emailB).toLowerCase();
+
+        if (sortA < sortB) {
```

Comment:
It's definitely not basic! String comparisons vary by language. In Javascript, it will do a letter-by-letter alphabetic comparison: `a` is less than `b`, `b` is less than `c`, etc. (Converting them to lowercase first gets rid of any weird things about whether `A` is also less than `b`).  So a string is "less than" another string if it'd come first when the strings were put in alphabetical order.